src/cpp/rtps/history/History.cpp

```diff
@@ -42,9 +42,17 @@ History::History(
     , mp_mutex(nullptr)
 
 {
+    int32_t max_caches = std::max(att.maximumReservedCaches, 0);
     int32_t initial_caches = std::max(att.initialReservedCaches, 0);
     m_changes.reserve(static_cast<size_t>(initial_caches));
 
+    PoolConfig pool_config
+    {
+        att.memoryPolicy, att.payloadMaxSize,
+        static_cast<uint32_t>(initial_caches),
+        static_cast<uint32_t>(max_caches)
```

Comment:
```suggestion
        att.memoryPolicy,
        att.payloadMaxSize,
        static_cast<uint32_t>(initial_caches),
        static_cast<uint32_t>(max_caches)
```
It makes the reading easier